src/three-components/ModelScene.js

```diff
@@ -105,6 +105,9 @@ export default class ModelScene extends Scene {
     // depthTest=false
     this.skysphere.renderOrder = 1;
 
+    // TODO: hook to transparent attribute
+    this.skysphere.visible = false;
```

Comment:
... would there need to be more logic around that? There are a few place that expect the skysphere to exist.